fhir_zulip_nlp/fhir_zulip_nlp.py

```diff
@@ -48,9 +48,10 @@
 # TODO: change CSV names to clarify that `zulip_raw_results.csv` is only for queried / matched messages
 RAW_RESULTS_MATCHES_FILENAME = 'zulip_raw_results.csv'
 RAW_RESULTS_ALL_FILENAME = 'zulip_raw_results_all.csv'
+STREAM_NAME_FIELD = 'display_recipient'  # strange, but this is the 'stream name' field from Zulip API
 CONFIG = {
     'zuliprc_path': os.path.join(ENV_DIR, '.zuliprc'),  # rc = "runtime config"
-    'streams': ['terminology'],
+    'streams': ['terminology', 'implementers', 'snomed'],
```

Comment:
@rohaher Here's where the different streams are declared.

Ideally I'd add this as an option in the google sheet. Maybe one day.